Realm/RLMRealm+Sync.h

```diff
@@ -18,7 +18,7 @@
 
 #import <Foundation/Foundation.h>
 
-#import "RLMRealm.h"
+#import <Realm/RLMRealm.h>
```

Comment:
Is this angled brace notation required now?

Reply:
Headers which are part of the framework umbrella header have to be imported via the framework path (and everything else needs to not be). This is a hard requirement for spm, and for non-spm cases not doing it was broken in ways that probably didn't matter (if someone had a file named RLMRealm.h in their project we'd import that rather than the framework's RLMRealm.h). Xcode now warns about bad imports in framework headers by default.